docs/business-finder-links-based-facets.md

```diff
@@ -0,0 +1,40 @@
+# EU Exit Business Readiness finder links-based facets
+
+This document explains how the [EU Exit Business Readiness Finder](https://www.gov.uk/find-eu-exit-guidance-business) ("business finder") uses [content-tagger](https://github.com/alphagov/content-tagger) 'links' to filter the finder and retrieve content from different publishing apps.
+
+## How is this different from the user perspective?
+
+In most finders, search results get more granular as you select values in different facets (the number of results goes down). The business finder search results get more _broad_ as you select values in different facets (number of results goes up). In addition, search results only show content from one publishing app.
+
+The business finder can be sorted by topic ("grouped display"). In this context, a 'topic' is the label of the facet (such as 'Personal data'), except for the 'Sector / Business area' facet, where the _facet value_ (such as 'Aerospace') is the topic. Search results can show content from multiple publishing apps.
+
+![Screenshot of 'Sort by Topic'](https://user-images.githubusercontent.com/93511/52811122-38eb2500-308c-11e9-843f-4df29b719235.png)
+
+This view only applicable to the business readiness finder, but is not yet a candidate for deletion, as it's been proven to be a useful feature.
+
+## How this works technically
+
+Standards finders use facets that have been published as part of the content item itself, which therefore appear under `details["facets"]` in the [content item JSON](https://www.gov.uk/api/content/cma-cases).
+
+The business finder uses facets that reference 'links' published by content-tagger, which therefore appear under `links["facet_group"]` in the [business finder content item JSON](https://www.gov.uk/api/content/find-eu-exit-guidance-business).
+
+Let's call these "details-based facets" and "links-based facets" respectively.
+
+| details-based facets | links-based-facets |
+|----------------------|--------------------|
+|![screenshot of content item JSON for /cma-cases](https://user-images.githubusercontent.com/5111927/60020202-6f166200-9687-11e9-86e9-1ba9020f0e27.png)|![screenshot of content item JSON for /find-eu-exit-guidance-business](https://user-images.githubusercontent.com/5111927/60020210-73db1600-9687-11e9-9cb8-e942964a27dc.png)|
+
```

Comment:
Everything below this point should be documented in content-tagger rather than here, as that's where knowing how the emails are triggered, and how the facets are published is useful.

Reply:
I'm not sure about that - the content below this point is quite coupled to why links-based facets were a good decision for the business finder. I don't think I could move it out to content-tagger without it losing its usefulness (as the version in content-tagger would need stripping down).

That sounds like an ADR - I'm happy to restructure the file as an ADR if that helps.